baseline-sim/src/mem_mapped.rs

```diff
@@ -81,14 +85,16 @@ pub const T1CR_ADDR: Addr = TIMER_MEM_MAPPED_BASE + TIMER_PIN_ADDRS * 1 + 0; //
 pub const T1DR_ADDR: Addr = TIMER_MEM_MAPPED_BASE + TIMER_PIN_ADDRS * 1 + 1; // xFE63
 
 pub const TIMER_BASE_INT_VEC: Addr = INTERRUPT_SERVICE_ROUTINES_START_ADDR + (TIMER_OFFSET as Addr); // x1E0;       // TODO: do this in a better way
-pub const T0_INT_VEC: u8 = 128 + GPIO_OFFSET + 0; // xE0
-pub const T1_INT_VEC: u8 = 128 + GPIO_OFFSET + 0; // xE1;
+pub const T0_INT_VEC: u8 = 128 + TIMER_OFFSET + 0; // xE0
+pub const T1_INT_VEC: u8 = 128 + TIMER_OFFSET + 0; // xE1;
```

Comment:
Should be `+ 1` I think

Reply:
Fixed!

Thought it would be better to resolve with the merge, tested it on my end and everything compiles fine with the changes.